{"name": "test_create_order_user_not_found", "status": "failed", "statusDetails": {"message": "AssertionError: assert 500 == 404\n +  where 500 = <Response [500]>.status_code", "trace": "valid_token = 'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJ6aGFuZ3NhbiIsImV4cCI6MTc0ODYxNjE4NX0.aKv55HpCiFLUUHaKMKwOmLEet1Z_DxB-ynl_nq2qI0w'\n\n    def test_create_order_user_not_found(valid_token):\n        url = f\"{BASE_URL}/orders/create\"\n        headers = {\"Authorization\": valid_token}\n        payload = {\n            \"address_id\": 1\n        }\n        response = requests.post(url, json=payload, headers=headers)\n>       assert response.status_code == 404\nE       assert 500 == 404\nE        +  where 500 = <Response [500]>.status_code\n\ntests/test_orders_create_post.py:47: AssertionError"}, "start": 1748586771369, "stop": 1748586771374, "uuid": "f63afb52-4829-4541-9714-873304bf542b", "historyId": "c3f5b6666e7249288df0ae15eade869c", "testCaseId": "c3f5b6666e7249288df0ae15eade869c", "fullName": "tests.test_orders_create_post#test_create_order_user_not_found", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_orders_create_post"}, {"name": "host", "value": "puxinyudeMacBook-Air.local"}, {"name": "thread", "value": "92108-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_orders_create_post"}]}